utils/rpms/daos.spec

```
@@ -335,6 +338,9 @@ getent group daos_admins >/dev/null || groupadd -r daos_admins
 %{_libdir}/*.a
 
 %changelog
+* Mon Jan 06 2020 Ravindran Padmanabhan <ravindran.padmanabhan@intel.com> - 0.8.0-2
```

```suggestion
* Mon Jan 06 2020 Ravindran Padmanabhan <ravindran.padmanabhan@intel.com> - 0.8.0-3
```

Update the release here.  Should probably also update the date since you are going to have to push a new commit anyway.

---

Updated.